Server/mods/deathmatch/logic/CScriptDebugging.cpp

```diff
@@ -143,16 +143,38 @@ void CScriptDebugging::PrintLog(const char* szText)
     }
 }
 
+bool CScriptDebugging::CheckForSufficientDebugLevel(unsigned int uiPlayerDebugLevel, unsigned int uiMessageDebugLevel)
+{
+    bool uiSufficientDebugLevel = false;
```

Comment:
Please don't use Hungarian notation, it's also wrong here. Rename the variable to `isSufficient` or similiar.